functions/influxdb+grafana.bash

```diff
@@ -33,20 +33,26 @@ influxdb_grafana_setup() {
 
   FAILED=0
   text_intro="This will install and configure InfluxDB and Grafana. For more information please consult this discussion thread:\\nhttps://community.openhab.org/t/13761/1\\n\\nNOTE for existing installations:\\n - Grafana password will be reset and configuration adapated\\n - If local installation of InfluxDB is choosen, passwords will be reset and config files will be changed"
-  text_lowmem="WARNING: InfluxDB and Grafana tend to use a lot of memory. Your machine reports less than 1 GB memory, and we STRICTLY RECOMMEND NOT TO PROCEED!\\n\\nDISCLAIMER: Proceed at your own risk and do NOT report tickets if you run in problems."
-  text_fail="Sadly there was a problem setting up the selected option. Please report this problem in the openHAB community forum or as a openHABian GitHub issue."
+  text_unsupported="You are trying to install Grafana on a SBC to have an oldish ARMv6l set instruction set only such as a RPi0W.\\nThis requires you to install a special package.\\n openHABian does not support that. You can still go for a manual install.\\n\\nGrafana software downloads are available at https://grafana.com/grafana/download?platform=arm"
```

Comment:
Variable should be camelCase and "You are trying to install Grafana on a SBC to have an oldish ARMv6l set instruction set only such as a RPi0W." Should be "You are attempting to install Grafana on a SBC  which has an older ARMv6l instruction set only, such as a RPi0W."